cardinal/ecs/persona.go

```diff
@@ -106,7 +121,8 @@ func buildPersonaTagMapping(wCtx WorldContext) (map[string]personaTagComponentDa
 				errs = append(errs, err)
 				return true
 			}
-			personaTagToAddress[sc.PersonaTag] = personaTagComponentData{
+			lowerPersona := strings.ToLower(sc.PersonaTag)
+			personaTagToAddress[lowerPersona] = personaTagComponentData{
 				SignerAddress: sc.SignerAddress,
 				EntityID:      id,
 			}
```

Comment:
you'll need to update the `buildPersonaTagMapping` function to have its keys be lowered as well

Reply:
done, the mapping only exists during a tick and so when we append to this map in `RegisterPersonaSystem` appending as a lower cased persona will make it so the next tick's check will be performed as expected